rated_list.md

```diff
@@ -131,27 +131,23 @@ def compute_node_score(rated_list_data: RatedListData,
 Function that is called whenever we get the peer list of a node.
 
 ```python
-def on_get_peers_response(rated_list_data: RatedListData, node: NodeId, children: Sequence[NodeId]):
+def on_get_peers_response(rated_list_data: RatedListData, child_id: NodeId, peers: Sequence[NodeId]):
```

Comment:
A little bit confused about the renaming here. Why `node` -> `child_id`? Woudn't it be the "parent" of the peers rather than the child?

Reply:
ah my bad. changed it. just wanted to change children to peers because the second for loop was confusing